### DISCUSSION TITLE
How to translate default fields like `date_created`, `user_created` once?

### DISCUSSION THREAD

**Discussion by loongmxbt** [directus/directus#15842] [Archive]
Hi, everyone, I have 30+ collections, and each collection has some default fields like `date_created`, `user_created` , `date_updated`, `user_updated`, `status`, etc.
How to translate default fields once across all collections?  So I don't have to edit 150+ times in one language translation.

**Comment by rijkvanzanten** [comment]
There's no global translation like that. Also FYI, those names aren't required to be used. The optional system fields inputs are editable 👍🏻
